eonx-com/easy-quality
# [ECS][Rector][PHPMD][PHPStan] Update quality tools

| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | yes
| Deprecations? | no
| Tests pass?   | yes    
| Fixed tickets | -   

BC: 
- some sniffers or rector have been deleted (by maintainers). 
- configs should be updated according documentations (ours and official)

Updates:
- Update ECS to the latest version
- Update custom sniffs and its tests to support latest ECS
- Update rector to the latest possible version
- Update rectors and its tests to support latest rector
- Update documentation
- Add sets
